tests/gen/gen_predefined.go

```diff
@@ -0,0 +1,202 @@
+// Copyright 2023 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package gen
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/sourcenetwork/defradb/client"
+	"github.com/sourcenetwork/defradb/client/request"
+)
+
+// GeneratePredefinedFromSDL generates documents for a schema from a predefined list
```

Comment:
nitpick:  I think you are missusing `schema` a few times in this documentation, for example  `for a schema from` should probably read `for the given collections from` or something.  Also `The schema is parsed ` is incorrect, the SDL doc is parsed, not the schema, same problem exists in `fields parsed from the schema.`.

Reply:
changed